### ISSUE TITLE
Allow requesting multiple specific validators from RPC

### ISSUE THREAD

**Issue by 0xKiwi** [prysmaticlabs/prysm#5104]
Currently within Prysm, there is `ListValidators ` for requesting all validators in a given epoch, and there is `GetValidator` for requesting one specific validator.

`ListValidators` should be changed to include a `indices` and `pubkeys` parameters, to let users request multiple indices in a single call.

**Comment by prestonvanloon** [comment]
Can you elaborate on the use case please?

**Comment by 0xKiwi** [comment]
If I want only want a subset or validators, but more than one. My only
option before this change was to request `GetValidator(i)` for every
validator in N. You can see the changes made to `ListValidators` in #5141.

**Comment by prestonvanloon** [comment]
@0xKiwi Are you sure? I just tried this and was able to retrieve 2 specific validators:

```
curl -X GET "https://api.prylabs.net/eth/v1alpha1/validators?indices=1&indices=2" -H "accept: application/json"
```

This is a `ListValidatorsRequest` in gRPC.

**Comment by 0xKiwi** [comment]
We can close this @prestonvanloon, it was fixed in #5141.
